src/test/java/org/nikita/chiken_bell/core/entity/CustomerTest.java

```diff
@@ -0,0 +1,41 @@
+package org.nikita.chiken_bell.core.entity;
+
+import org.junit.jupiter.api.Test;
+import org.nikita.chiken_bell.core.exception.PhoneEmptyException;
+import org.nikita.chiken_bell.core.exception.PhoneIncorrectException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CustomerTest {
+
+    @Test
+    void testInitCustomerWithCorrectPhone(){
```

Comment:
Please use constants for names, phones, addresses

Reply:
What is it means

Reply:
?